### DISCUSSION TITLE
Azure.PublicIP.Rule Query

### DISCUSSION THREAD

**Discussion by paulmccrady** [Azure/PSRule.Rules.Azure#2696] [Q&A] [answered]
We have a subscription with two pips in it.
Both are in the same region (UKSouth) and both have Availability zones 1, 2 and 3 configured.
One though is associated to kubernetes and resides in the mc resource group
We get PSRule erroring with

The public IP (kubernetes-af36bf06d27f745dcbab5d9f6c515124) deployed to region (uksouth) should be zone-redundant.

Is this an issue with the rule ?

**Comment by BernieWhite** [comment, marked as answer]
@paulmccrady Thanks for flagging this. I think the rule is fine but there may be an issue with the export process. Can I confirm:

- That you are seeing this based on data that has been exported from Azure using the **in-flight** option.
- If you have access to the JSON export can you confirm that the `zones` property is missing from the resource export.

If so, let me log a bug.

**Reply by paulmccrady** [reply]
@BernieWhite we use Export-AzRuleData with only the -OutputPath option.
You are correct, the zones proprty is missing for both ip addresses.
I've realised also that the other pip error was being suppressed so just confirming its an issue for both and not just the one created for AKS

**Reply by BernieWhite** [reply]
Logged as #2698

**Reply by BernieWhite** [reply]
This will ship in the next minor release v1.34.0.
